sale_layout_category_hide_detail/static/src/css/invoice_report_style.scss

```diff
@@ -0,0 +1,3 @@
+span.float-right {
```

Comment:
This looks like a patch to something else having side effects :thinking:

Reply:
Thanks for the review, I reduced the css to 0 this time updating to the bootstrap new classes.